### DISCUSSION TITLE
Flight testing question

### DISCUSSION THREAD

**Discussion by anoopk18** [bitcraze/discussions#523] [Q&A] [answered]
Hi,

I was wondering if anyone has performed flight testing in hover and forward flight to determine average rpm for the rotors of a crazyflie. I looked at this: [https://www.bitcraze.io/2015/02/measuring-propeller-rpm-part-2/], but it looks like 19000 rpm is with RPM measurement board attached. I know that conventional drones fly at an average of 6000 rpm, but crazyflie is much smaller and lighter.

Additionally, is there a minimum height clearance that it can be flow before starting to experience significant ground effects that could throw of one from its trajectory?

**Comment by tobbeanton** [comment, marked as answer]
I think you observation is pretty correct, with no additional decks it hovers at around 17-18k RPM. For more information and details you can check [this](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/pwm-to-thrust/) documentation.

I don't have an exact number for the ground clearance but somewhere around 10cm I would say.
